action.yml

```diff
@@ -0,0 +1,94 @@
+name: Sync Templates
+description: Synchronizes Templates between your repository and Fiberplane
+branding:
+  icon: tag
+  color: gray-dark
+
+inputs:
+  api-token:
+    description: API token used to access the Fiberplane API with
+    required: true
+  workspace-id:
+    description: ID of the workspace to which the templates should be uploaded to
+    required: true
+  fp-base-url:
+    description: Base URL of the Fiberplane API
+    default: https://studio.fiberplane.com
+  templates-directory:
+    description: "Custom directory that should be monitored for Template JSONNET files (default: .fiberplane/)"
+    default: .fiberplane/templates/
```

Comment:
Maybe we should allow the user to specify a glob. And then make the default be `.fiberplane/templates/*.jsonnet`. I think that the github actions sdk has some support for it.